### PR TITLE
replace anbox with waydroid

### PR DESCRIPTION
**Pull request by Fuseteam**

### CODE REVIEW

**Comment by doniks** [comment]
Hey @Fuseteam looks good. I see it's draft so no pressure at your own time, but I had a quick look and I wonder whether we should *replace* anbox with waydroid or just *add* it. An honest question, I don't know either of those two more than by name, so I'm just asking. 

One little bit of nit picking please make sure the headline underlines eventually have the same length as the headlines themselves and maybe you have seen right now it fails to build: https://ci.ubports.com/blue/organizations/jenkins/docs.ubports.com/detail/PR-561/1/pipeline/#step-29-log-116

**Comment by Fuseteam** [comment]
Thanks for the  quick feedback! I appreciate it!

I think we should replace anbox with waydroid for a few reasons:
1. Anbox is now officially deprecated according to anbox.io
2. Anbox has been removed from the rootfs starting ota-25 (if memoru serves)
3. Focal has replaced anbox with waydroid

**Comment by doniks** [comment]
Ah ok cool.thanks for the anbox clarification. Anbox is dead, long live waydroid 👍

**Comment by Fuseteam** [comment]
hmmm how do i redirect it :thinking:

**Comment by Fuseteam** [comment]
aha!

**Comment by doniks** [comment]
Hi @Fuseteam this is still marked as draft. Just checking whether this is intentional and you're still working on this, or it is ready for review?

File: userguide/dailyuse/waydroid.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+.. note::
+    You now have an ADB server running on your Ubuntu Touch device.
+
+How to install new APKs
+-----------------------
+
+- Copy the APK to ``/home/phablet/Downloads``. Then run the following from your computer::
+
+    adb shell # connect from your computer to your Ubuntu Touch device you can also simply open de terminal app on your Ubuntu Touch device
```
Comment (ElectrodeYT) [inline userguide/dailyuse/waydroid.rst:30]: The comment has some mistakes in it, would probably be better as
`adb shell # Connect to your Ubuntu Touch device. You can also simply open the terminal app on your device.`

File: userguide/dailyuse/waydroid.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    waydroid app install /home/phablet/Downloads/my-app.apk #
```
Comment (ElectrodeYT) [inline userguide/dailyuse/waydroid.rst:31]: Worlds tiniest nit-pick, but there is a stray # here

File: userguide/dailyuse/waydroid.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+- F-Droid: https://f-droid.org/
+- ApkTrack: https://f-droid.org/packages/fr.kwiatkowski.ApkTrack/
+
+How to uninstall apps
+---------------------
+
+- To uninstall apps from the Ubuntu Touch device, run ``adb uninstall [APP_ID]`` from your computer::
+
+    adb shell # connect from your computer to your Ubuntu Touch device, you can also simply open de terminal app on your Ubuntu Touch device
```
Comment (ElectrodeYT) [inline userguide/dailyuse/waydroid.rst:48]: Same as above

File: userguide/dailyuse/waydroid.rst [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    waydroid app remove [APP_ID] # you can find the APP_ID by using waydroid app list
```
Comment (ElectrodeYT) [inline userguide/dailyuse/waydroid.rst:49]: Maybe put `waydroid app list` in quotes?

File: userguide/dailyuse/waydroid.rst [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    ln -s ~/.local/share/waydroid/data/media/0/Documents ~/Documents/android
+    ln -s ~/.local/share/waydroid/data/media/0/Pictures ~/Pictures/android
+    ln -s ~/.local/share/waydroid/data/media/0/Music ~/Music/android
+    ln -s ~/.local/share/waydroid/data/media/0/Movies ~/Videos/android
+
+
+Troubleshooting
+---------------
+
+- When you want to install an APK, but get the error ``Invalid APK file`` that error could also mean "file not found"
```
Comment (ElectrodeYT) [inline userguide/dailyuse/waydroid.rst:70]: This is phrased weirdly. Maybe something such as "Error `Invalid APK file` when trying to install an app"?

**Review by ElectrodeYT — CHANGES_REQUESTED** [review]

**Comment by Fuseteam** [comment]
Hi sorry hadn't checked github in a while, lemme dubble check

File: userguide/dailyuse/waydroid.rst (hunk 1 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst:30]: fixed

File: userguide/dailyuse/waydroid.rst (hunk 2 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst:31]: fixed

File: userguide/dailyuse/waydroid.rst (hunk 3 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst:48]: fixed

File: userguide/dailyuse/waydroid.rst (hunk 4 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst:49]: fixed

File: userguide/dailyuse/waydroid.rst (hunk 5 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst:70]: changed phrasing

**Comment by Fuseteam** [comment]
i feel like i forgot something i was going to do.........but i guess we'll cross that bridge when we get there

File: userguide/dailyuse/waydroid.rst [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+`Waydroid <https://waydro.id>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as Ubuntu Touch.
+
+.. note::
+    "Computer" refers to another device you connect your Ubuntu Touch device to (via USB here).
+    Your USB-attached computer must have ``adb`` and ``fastboot`` installed.
+
+.. note::
+    You will need to execute commands on your Ubuntu Touch device (and/or attached computer) to initialize Waydroid and install APKs.
+    You can do that by using the terminal application, but it is easier to use ``adb shell`` or :doc:`set up SSH </userguide/advanceduse/ssh>` to access your Ubuntu Touch device from your computer.
```
Comment (doniks) [inline userguide/dailyuse/waydroid.rst:12]: please add link for our adb page as well

File: userguide/dailyuse/waydroid.rst [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+
+How to set up
+-------------
+
+All Halium 9 and newer devices ship with waydroid out of the box starting Ubuntu Touch focal, if your device fits these requirements you can follow the folowing steps
```
Comment (doniks) [inline userguide/dailyuse/waydroid.rst:18]: starting *with* Ubuntu Touch focal

Comment (doniks) [inline userguide/dailyuse/waydroid.rst:18]: folowing -> following

File: userguide/dailyuse/waydroid.rst [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+#. Run ``adb shell`` from your computer to get a shell on your Ubuntu Touch device (or open the terminal app on your ubuntu touch device).
```
Comment (doniks) [inline userguide/dailyuse/waydroid.rst:20]: Maybe rephrase to "Open a shell on your Ubuntu Touch device (adb, ssh or terminal app)" or similar

File: userguide/dailyuse/waydroid.rst [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+#. Run the following command on your Ubuntu Touch device: ``waydroid init``.
+
+Now you're done! You might have to refresh the app drawer (pull down from the center of the screen and release) for the new Android apps to show up.
+
+How to install new APKs
+-----------------------
+
+- Copy the APK to ``/home/phablet/Downloads``. Then run the following from your computer::
```
Comment (doniks) [inline userguide/dailyuse/waydroid.rst:28]: I'd suggest to add a separate bullet point, for the "open a shell....", I think it's best to just copy paste from above, you know keep the same pattern, makes it easier to read. and then a separate bullet point for waydroid app install. No need for exit I think. Again keep the same pattern as above

Comment (doniks) [reply userguide/dailyuse/waydroid.rst:28]: ah. also, please pick either enumeration or itemization and use consistently for both lists. And either make the "Done... refresh .. pull" sentence in both cases part of the list or not part of the list

File: userguide/dailyuse/waydroid.rst [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+- To keep your apps up to date you can use of F-Droid or Amazon Appstore. If you want to install any of the above apps you can find them here:
+
+- F-Droid: https://f-droid.org/
+- Amazon Appstore: https://www.amazon.com/gp/mas/get-appstore/android
+
+How to uninstall apps
+---------------------
+
+- To uninstall apps from the Ubuntu Touch device, run ``adb uninstall [APP_ID]`` from your computer::
```
Comment (doniks) [inline userguide/dailyuse/waydroid.rst:47]: aaand also here, please use the same style for this set of instructions as the two above

File: userguide/dailyuse/waydroid.rst [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+- Copy the APK to ``/home/phablet/Downloads``. Then run the following from your computer::
+
+    adb shell # Connect to your Ubuntu Touch device. You can also simply open the terminal app on your device.
+    waydroid app install /home/phablet/Downloads/my-app.apk
+    exit
+
+- Done! You might have to refresh the apps scope (pull down from the center of the screen and release) for the new Android apps to show up.
+
+Keep your apps up to date
+-------------------------
```
Comment (doniks) [inline userguide/dailyuse/waydroid.rst:37]: This part confuses me a bit. We promote apk's for install, but once I have installed an app store like fdroid or amazon, I could also use that app store to *install* further apps, right. Which in fact is more convenient than the apk route ... How about changing "Keep your apps up to date" to "Use an app store in waydroid" and then explain that you could use one of the two and that would allow you the installation *and* update of apps

**Review by doniks — CHANGES_REQUESTED** [review]

File: userguide/dailyuse/waydroid.rst (hunk 11 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst:37]: Oh yeah true, will update

**Comment by Fuseteam** [comment]
Crud forgot my gpgkey, will fix

File: userguide/dailyuse/waydroid.rst (hunk 6 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst]: added

File: userguide/dailyuse/waydroid.rst (hunk 7 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst]: fixed

File: userguide/dailyuse/waydroid.rst (hunk 9 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst]: made it more consistent

File: userguide/dailyuse/waydroid.rst (hunk 11 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst:33]: uodated and elaborated

File: userguide/dailyuse/waydroid.rst (hunk 10 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst]: made it more consistent

File: userguide/dailyuse/waydroid.rst (hunk 8 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst]: rephrased

File: userguide/dailyuse/waydroid.rst (hunk 7 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst]: fixed

File: userguide/dailyuse/waydroid.rst [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+
+`Waydroid <https://waydro.id>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as Ubuntu Touch.
+
+.. note::
+    "Computer" refers to another device you connect your Ubuntu Touch device to (via USB here).
+    Your USB-attached computer must have ``adb`` and ``fastboot`` installed.
+
+.. note::
+    You will need to execute commands on your Ubuntu Touch device (and/or attached computer) to initialize Waydroid and install APKs.
+    You can do that by using the terminal application, but it is easier to use :doc:``adb shell </userguide/advanceduse/adb>`` or :doc:`set up SSH </userguide/advanceduse/ssh>` to access your Ubuntu Touch device from your computer.
```
Comment (doniks) [inline userguide/dailyuse/waydroid.rst:12]: cool. I see just one more thing. the adb link doesn't work https://ci.ubports.com/job/docs.ubports.com/job/PR-561/11/artifact/_build/html/userguide/dailyuse/waydroid.html I think it misses a backtick

**Review by doniks — CHANGES_REQUESTED** [review]

File: userguide/dailyuse/waydroid.rst (hunk 12 above)
Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst:12]: darn it, i'll take a look

Comment (Fuseteam) [reply userguide/dailyuse/waydroid.rst]: it appears i had too many backticks

**Review by doniks — APPROVED** [review]
